_posts/2015-03-31-how-we-do-css.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: How We Do CSS at Zumba
+description: This is how we do CSS at Zumba.
+tags: [technology, css, sass, front end, engineering]
+author: neptunz
+---
+
+As you may or may not know Zumba's website consists of a robust app that includes a fully custom ecommerce shop that supports international transactions, class and training search and registration, Zumba Instructor Network (ZIN) admin portal and many other apps. Our in-house tech department is responsible for managing the interactive products, developing, testing and deploying the projects.
+
+I was inspired by the teams at <a href="http://dev.ghost.org/css-at-ghost/" target="_blank">Ghost</a> and <a href="https://github.com/styleguide/css" target="_blank">Github</a> to write this article to give you an idea of how we do CSS at Zumba.
+
+
+### Intro - Browser and Device Support
+We try to keep our code as light, modular and DRY as possible. We use Modernizr, Autoprefixer and some Foundation components. We try to keep our nesting no more than 4 levels deep and we try to make sure that  our naming convention lends itself to reusable components.
+
+We support the last few versions of popular browsers (Chrome, Safari and Firefox). At the time of writing this we currently support IE 8+. About 2% of our users still use IE 8; we are not out of the woods yet.
+
+We made the leap to a responsive application with our recent redesign of the site. Therefore, we support a wide range of desktop, tablet and mobile devices. At the time of writing this we don’t currently support smart watches. We will soon, though… so, "watch" out for that... sorry.
+
+
+### Preprocessor
+We use Sass with Compass as our preprocessor. Our team is proficient and has much training on this popular pre-processor and it’s large following helps.
+
+
+### Folder/Sandwich Structure
+Here’s the fun stuff - we structured our css based on a few important factors:
+
+
+#### + Modularity and Reusability
+Develop components that can be reused throughout the application.
+
+
+#### + CSS Rules
+Consider the size of the compiled stylesheets per section of the site to not exceed IE9’s rules limit of 4,095. Our previous site was compiled into two massive stylesheets and we learned the hard way. We currently break it down per section of the site. See examples below.
```

Comment:
add 'in order' in front of "to not exceed IE9’s rules limit of 4,095"
add 'that' in front of 'the hard way'
this sentence - "We currently break it down per section of the site" is redundant since you already mention that the stylesheets are broken down per section of the site.